notebooks/5_add_columns.ipynb

```diff
@@ -0,0 +1,727 @@
+{
```

Comment:
<p>This seems to include a lot of detail that may be distracting for the reader. Perhaps just make a simple statement about the conversion factor?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>